pages/migrations/short_codes.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.1 on 2016-03-08 16:59
+from __future__ import unicode_literals
+
+from django.db import migrations
+from wagtail.wagtailredirects.models import Redirect
+from django.db import connection
+import os
+migration_path = os.path.dirname(os.path.realpath(__file__))
+
+
+def add_osc_shortcodes(*args, **kwargs):
```

Comment:
This is not the proper place for this. I think there is confusion because we are calling these migraitons - this directory is reserved for Django generated migrations. I think you should move this to a [management command](https://docs.djangoproject.com/en/1.9/howto/custom-management-commands/) - so it can be run from manage.py when it's required.